library/core/src/clone.rs

```diff
@@ -140,7 +140,7 @@ pub trait Clone: Sized {
 /// Derive macro generating an impl of the trait `Clone`.
 #[rustc_builtin_macro]
```

Comment:
also, what's the difference between this attr and the one you're adding?

Reply:
all of the macros you've tagged with this new attribute are already tagged with this one -- is there a distinction that this new macro is making?

Reply:
This is specifically for derive macros - if we just checked `rustc_builtin_macro`, I believe illegal references could be made in `println!()` etc.
I didn't see this before though - maybe checking for *both* `automatically_derived` and `rustc_builtin_macro` would be a quicker solution to this instead of a new attr.